packages/client/src/upload.ts

```diff
@@ -87,6 +96,27 @@ export async function* upload(
 
         const { data } = file;
 
+        uploadProgress.bytesUploaded = 0;
```

Comment:
**suggestion (non-blocking):** remove?

```suggestion
```

Is this necessary? The default value is already `0`, right?

Reply:
This is in case the file upload fails and is being retried. So it's a reset of the bytes uploaded counter.